.github/workflows/pr-realm-js.yml

```diff
@@ -108,6 +108,17 @@ jobs:
         if: ${{ runner.os == 'Windows' }}
         uses: ilammy/msvc-dev-cmd@v1
 
+      - name: Setup CMake
+        if: ${{ runner.os == 'Windows' }}
```

Comment:
Might make sense to do this on all platforms to be resilient to regressions on other platforms?
```suggestion
```

Reply:
If it ain't broken, don't fix it :smile: